container-search/src/main/java/com/yahoo/search/dispatch/LoadBalancer.java

```diff
@@ -52,7 +52,7 @@ public LoadBalancer(SearchCluster searchCluster, boolean roundRobin) {
      * @param rejectedGroups if not null, the load balancer will only return groups with IDs not in the set
```

Comment:
You need to remove the javadoc for query param to make the build pass.

Reply:
Done